src/pages/HomeAdministrator.vue

```diff
@@ -1,32 +1,61 @@
 <template>
   <main class="container main">
     <section class="main-body">
-      <PvPanel header="Your administrations">
-        <template #icons>
-          <label class="mr-2" for="dd-sort">Sort by</label>
-          <PvDropdown
-            v-model="sortKey"
-            input-id="dd-sort"
-            :options="sortOptions"
-            option-label="label"
-            data-cy="dropdown-sort-administrations"
-            @change="onSortChange($event)"
-          />
-        </template>
+      <div>
+        <div class="flex flex-wrap align-items-center justify-content-between">
+          <div class="flex-column flex-wrap gap-2">
+            <div class="text-3xl font-bold text-gray-600">Your Administrations</div>
+          </div>
+          <div class="flex gap-3 align-items-center justify-content-start p-3">
+            <div class="flex flex-column gap-1">
+              <small id="search-help" class="text-gray-400">Search by administration name</small>
+              <div class="flex align-items-center">
+                <PvInputGroup>
+                  <PvInputText
+                    id="search"
+                    v-model="searchInput"
+                    @keyup.enter="onSearch"
+                    placeholder="Search administrations"
+                  >
```

Comment:
Could we use Primevue's autocomplete component (https://primevue.org/autocomplete/) to suggest things as the user types?